src/openvic-simulation/dataloader/Dataloader.hpp

```diff
@@ -70,6 +77,8 @@ namespace OpenVic {
 		 * DAT-24
 		 */
 		fs::path lookup_file(std::string_view path, bool print_error = true) const;
+		/* Checks alternate file endings, e.g. if "*.tga" doesn't exist then try "*.dds" */
```

Comment:
This should potentially be renamed/extended, being able to pass multiple allowed file extensions seems useful generally.

Reply:
I've renamed it to `lookup_image_file_or_dds`, originally it used a list of potential endings but I simplified it once I realised the only case that needs changing (in vanilla at least) is `*.tga` -> `*.dds`. The function doesn't actually check if the input is `*.tga`, if you pass it `*.dds` it'll attempt to look it up twice.
I'll leave the implementation as is for now, we can decide whether to treat this as a special case to account for mistakes in the defines, or whether we want to support fuzzy image format for GFX asset loading more broadly (`*.dds` -> `*.tga` as well as other image formats) so that UI modders don't have to worry about preserving the (often archaic) image formats or have to modify the GFX files if all they're changing is the contents of an asset.